src/common/dockerUtil.ts

```diff
@@ -65,6 +66,28 @@ const isDockerInstalled$ = (): Observable<boolean> => {
   );
 };
 
+const isDockerComposeInstalled$ = (): Observable<boolean> => {
+  return execCommand$(AVAILABLE_COMMANDS.IS_INSTALLED_COMPOSE).pipe(
+    map((output) => {
+      if (output.includes("docker-compose version")) {
+        return true;
+      }
+      return false;
+    }),
+    catchError((e: any) => {
+      if (
+        e.message?.includes(
+          "'docker' is not recognized as an internal or external command"
```

Comment:
When executing `docker-compose version` on an Ubuntu machine without docker-compose, I get a response starting with `Command 'docker-compose' not found`.